src/WebJobs.Extensions.DocumentDB/Trigger/DocumentDBTriggerLeaseOptions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
+{
+    using System;
+    using Microsoft.Azure.Documents.ChangeFeedProcessor;
+
+    /// <summary>
+    /// Configuration options for the Leases used by the trigger mechanism
+    /// </summary>
+    public class DocumentDBTriggerLeaseOptions
```

Comment:
It should be fine to expose that -- we do something similar for other bindings (we expose `EventProcessorOptions` and `PartitionManagerOptions` in Event Hubs, for example). It'd reduce the wrapper code a little.